cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -50,11 +53,52 @@ var (
 		"How often scale down possiblity is check")
 )
 
+var (
```

Comment:
Can you please move metrics to a separate file (or even package)?
As an example:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/metrics/metrics.go